test/flows.spec.js

```diff
@@ -379,7 +383,16 @@ describe('Flows', function() {
         expect(response.path).to.equal('/v1/charges');
         expect(response.request_id).to.match(/req_[\w\d]/);
         expect(response.status).to.equal(402);
-        expect(response.elapsed).to.be.within(50, 30000);
+        expect(response.elapsed).to.equal(
+          response.request_end_time - response.request_start_time
+        );
+        expect(response.request_end_time).to.be.within(
+          lowerBoundStartTime,
```

Comment:
sorry, tiny nit: probably a bit cleaner to use `response.request_start_time` here and then remove the `expect` below.

Reply:
I went ahead and updated. This should be good for you to merge now.